Formula/sqlcipher.rb

```diff
@@ -34,13 +34,27 @@ def install
     ]
 
     if build.with?("fts")
-      args << "CFLAGS=-DSQLITE_HAS_CODEC -DSQLITE_ENABLE_FTS3 -DSQLITE_ENABLE_FTS3_PARENTHESIS -DSQLITE_ENABLE_FTS5"
+      args << "CFLAGS=-DSQLITE_HAS_CODEC -DSQLITE_ENABLE_JSON1 -DSQLITE_ENABLE_FTS3 -DSQLITE_ENABLE_FTS3_PARENTHESIS -DSQLITE_ENABLE_FTS5"
```

Comment:
If it's enabled in both the if and the else, then it belongs in the array above :)

Reply:
Oops.  Good point.  Will fix shortly, thanks. :smile: